questdb-rs/src/ingress/mod.rs

```diff
@@ -26,12 +26,17 @@
 //!
 //! The `ingress` module implements QuestDB's variant of the
 //! [InfluxDB Line Protocol](https://questdb.io/docs/reference/api/ilp/overview/)
-//! (ILP) over TCP.
+//! (ILP) over HTTP and TCP. We recommend using HTTP, and consider TCP as a
+//! legacy protocol variant.
```

Comment:
I think it's too early to call TCP legacy.
Rather, I think it should state why HTTP.

Something like "We recommend using the new ILP/HTTP protocol since it provides better error feedback and fine transaction control.".